### DISCUSSION TITLE
Honest question, how safe is this compared to AC?

### DISCUSSION THREAD

**Discussion by richardotomislav** [seerge/g-helper#335] [Q&A]
Good morning, don't get me wrong with this question. I'm very interested in the program, and I'm gonna try it. 
But I want to know if there are any side effects that could happen if I use this software instead of AC. 

I'm sure there's nothing wrong with using this software but I want to be 100% sure. 

Many thanks.

**Comment by tzukiwy** [comment]
it is safe. you can view the source code yourself and build your own if you feel like it!

**Reply by richardotomislav** [reply]
> it is safe. you can view the source code yourself and build your own if you feel like it!

You know, I actually didn't think of that for some reason. Sometimes I question myself.
So it shouldn't do any side-effects gotcha.  

Thanks for the input.

**Comment by richardotomislav** [comment]
> Good morning, don't get me wrong with this question. I'm very interested in the program, and I'm gonna try it. But I want to know if there are any side effects that could happen if I use this software instead of AC.
> 
> I'm sure there's nothing wrong with using this software but I want to be 100% sure.
> 
> Many thanks.

Also, I forgot to ask, I'm not doing a full Windows refresh right now due to circumstances. Can I use the program with armory crate installed to test it and uninstall it when I can?

**Reply by seerge** [reply]
Yes, you can run it along armoury
